packages/framework/src/Resources/styles/admin/core/form/input.less

```diff
@@ -18,8 +19,8 @@
     width: @input-width;
     max-width: 100%;
     height: @input-height;
-    line-height: @input-line-height;
```

Comment:
Can you delete the variable definition? Are we using it somewhere else?

Reply:
Yes, we can. Deleted, thank you for your good point